### PR TITLE
Remove env, default urls, chainId

### PR DESCRIPTION
**Pull request by woomurf**
- Remove env (NODE_ENV) and default urls (AINFT_SERVER_ENDPOINT, AIN_BLOCKCHAIN_ENDPOINT), chainId.

### CODE REVIEW

File: src/event.ts [hunk 1]
```diff
@@ -37,8 +37,7 @@ export default class Event extends AinftBase {
       endAt,
       platform,
     };
-    const trailingUrl = '';
-    return this.sendRequest(HttpMethod.POST, trailingUrl, body);
+    return this.sendRequest(HttpMethod.POST, body);
```
Comment (woomurf) [inline src/event.ts:40]: Fixed this case

Comment (woomurf) [reply src/event.ts]: No.. reverted...

**Review by liayoo — APPROVED** [review]
LGTM, thanks for fixing this!!

**Comment by woomurf** [comment]
Thanks for review! merging now!

**Pull request merged**